exit_code: 1
args: --reload tests/async_error.ts
check_stderr: true
output: tests/async_error.ts.out
